From 07a9c5e53be47ff720d26cdcc0d4c8eb92f36c16 Mon Sep 17 00:00:00 2001
From: Kazunori Kobayashi <kkobayas@igel.co.jp>
Date: Tue, 24 Apr 2012 14:29:51 +0900
Subject: [PATCH 13/97] dfb-example: change a default value of queue-size for
 v4l2src

This change is necessary in continuous capturing with VIN driver.
---
 ext/directfb/dfb-example.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ext/directfb/dfb-example.c b/ext/directfb/dfb-example.c
index 02c9fa1..68d81b1 100644
--- a/ext/directfb/dfb-example.c
+++ b/ext/directfb/dfb-example.c
@@ -69,7 +69,7 @@ main (int argc, char *argv[])
 
   memset (&rect, 0, sizeof (rect));
   layer_id = 0;
-  queue_size = 4;
+  queue_size = 5;
   top = bottom = left = right = 0;
 
   opterr = 0;
-- 
1.7.10.4

